arches_lingo/src/arches_lingo/components/generic/ControlledListItem.vue

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+const UPDATE = "update";
```

Comment:
yeah please roll back the declaring of emit values as constants. Much rather _not_ have the two script tags